kokoro/presubmit.sh

```diff
@@ -5,21 +5,28 @@ set -o xtrace
 
 gcloud components install docker-credential-gcr
 
+# Docker service does not run by default in Big Sur but can be started with the following commands.
+if [ "${KOKORO_JOB_CLUSTER}" = "MACOS_EXTERNAL" ]; then
+docker-machine ls
+docker-machine start default
+export DOCKER_IP="$(docker-machine ip default)"
+echo $DOCKER_IP
+docker-machine ssh default "echo '{ \"insecure-registries\":[\"$DOCKER_IP:5000\", \"$DOCKER_IP:8080\"] }' | sudo tee /etc/docker/daemon.json "
+docker-machine ssh default "echo 'DOCKER_OPTS=\"--config-file=/etc/docker/daemon.json\"' | sudo tee -a /var/lib/boot2docker/profile "
+docker-machine ssh default "sudo /etc/init.d/docker restart"
+docker-machine env default
+eval "$(docker-machine env default)"
+
```

Comment:
Optional: might want to extract all this into a helper shell script. Or let's do it whenever we have to change anything about it, since it shows up twice.